### PR TITLE
media url added to answer, save benchmark model, classifier timeout 

### PR DESCRIPTION
**Pull request by j2l**
# Pull Request Template

## PR Checklist

- [ ] I have run `npm test` locally and all tests are passing.
```
lib\nlp\nlp-manager.js
  344:16  error  Unexpected string concatenation    prefer-template
```
I don't know "templates".

`  345:14  error  Unnecessary 'else' after 'return'  no-else-return`
as in lib\ner\ner-manager.js  254:12

- [ ] I have added/updated tests for any new behavior.
Don't know I to do it

- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: N/A

## PR Description
Adding a media url field to the answer to be able, later, to reply with some picture, audio in addition to text. Exemple: Barcelona + pic of Sagrada Família.

If this idea is validated, media should be present in result, outside the answer (result.media). This might change the format, so I won't touch it without approbation.

Addtionally, url could be used to gather information to pass to the asnwer (with a timeout).
Using `result.media`, `result.mediaGETheader`, `result.usemedia`
Exemple:
```
manager.addAnswer(
'en',
'travel',
'You want to travel from {{ fromCity }} to {{ toCity }} {{ date }}',
'',
'https://pubapi.atadi.vn/air/test/search/{{fromCity}}/{{toCity}}/{{formatteddate}}/100',
'header Authorization: NNNNN',
true
};
```
See https://documenter.getpostman.com/view/55211/atadi-public-api/RW1emdSE#929e7b76-fd97-4dd9-b5a1-cf8b2b8179d3
implying the need to add header and auth information in answer.

Feel free to rewrite/adapt.

### CODE REVIEW

**Comment by jesus-seijas-sp** [comment]
It has some prettier issues. Most of them can be fixed with npm run lintfix: https://travis-ci.com/axa-group/nlp.js/jobs/154625398

**Comment by j2l** [comment]
Thank you! didn't know this one. Fixed.
Do you want me to request it again or do you prefer to rewrite these additions? It's not great as is.

**Comment by jesus-seijas-sp** [comment]
You can run locally "npm run lint" or "npm test". 
Or push code and will be executed by travis, if you click in "Details" on the Travis check, and there you click "The build", will go to your build in different node versions, and each node version you can see the console with errors.

**Comment by j2l** [comment]
Learned some stuff today :) Thanks!
Fixed all the picky stuff from Travis however `npm run lint` shows other errors on files I didn't touch.
And I learned what it called "template", I'm using it without knowing its name.

Still failing at  test/recognizer/recognizer.test.js (17.378s)
      ✕ It should use context if conversation id is provided (2247ms)
Not sure if it's realted to any of my changes.

File: lib/nlp/nlp-excel-reader.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ class NlpExcelReader {
         row.language,
         row.intent,
         row.response,
+        row.url,
```
Comment (jesus-seijas-sp) [inline lib/nlp/nlp-excel-reader.js]: row.url should go after row.condition given the interface of the function. That's what cause the error in tests.

**Comment by jesus-seijas-sp** [comment]
I've add a comment where the error is. 
Also, I wish to understand better what is url/media, perhaps you can explain or put some docs...

File: lib/nlp/nlp-excel-reader.js (hunk 1 above)
Comment (j2l) [reply lib/nlp/nlp-excel-reader.js]: ok, fixed it. thanks!

**Comment by j2l** [comment]
Do I need to do something on my side?

**Comment by j2l** [comment]
Should I close/forget it?

**Comment by j2l** [comment]
Hmm, it was not ready to merge, more to see and adapt according to your goal. Might go on a branch.
As is, answer as an extra "-", please see https://coveralls.io/builds/19824783/source?filename=lib%2Fnlp%2Fnlp-manager.js#L348
If you'd like to rewrite it, please feel free. I CC0 this PR code 😄

**Comment by jesus-seijas-sp** [comment]
Well, the code is fine as it is declared as an optional parameter. And of course you can improve it and PR again with changes.

**Pull request merged**